modules/nf-core/crisprcleanr/normalize/main.nf

```diff
@@ -56,10 +56,6 @@ process CRISPRCLEANR_NORMALIZE {
     version_file_path <- "versions.yml"
     version_crisprcleanr <- paste(unlist(packageVersion("CRISPRcleanR")), collapse = ".")
     f <- file(version_file_path, "w")
-    writeLines('"${task.process}":', f)
-    writeLines("    crisprcleanr: ", f, sep = "")
-    writeLines(version_crisprcleanr, f)
-    close(f)
```

Comment:
Is this correct @LaurenceKuhl ?

Context: we are updating the template to v2.8 of nf-core/tools and this was automatically removed when I updated the module, not sure if it's OK or we should keep it.